swagger2.0/oic.r.vodlist.swagger.json

```diff
@@ -0,0 +1,135 @@
+    }
+  },
+  "schemes": ["http"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/VODListResURI" : {
+      "get": {
+        "description": "This resource describes the VODs that have been onboarded on the Bridge Platform.\n",
+        "parameters": [
+          {"$ref": "#/parameters/interface"}
+        ],
+        "responses": {
+            "200": {
+              "description" : "",
+              "x-example":
+                {
+                  "rt":    ["oic.r.vodlist"],
+                  "vods":  [
+                    {
+                      "n": "Window shatter sensor",
+                      "di": "54919CA5-4101-4AE4-595B-353C51AA1234",
+                      "econame": "Z-Wave"
+                    },
+                    {
+                      "n": "Door Lock",
+                      "di": "54919CA5-4101-4AE4-595B-353C51AA5678",
+                      "econame": "Zigbee"
+                    }
+                  ]
+                },
+              "schema": { "$ref": "#/definitions/vodlist" }
+            }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "interface" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.r", "oic.if.baseline"]
+    }
+  },
+  "definitions": {
+    "vodentry" : {
+      "description": "Information of a VOD created by the Bridge",
+      "type": "object",
+      "properties": {
+        "n": {
+            "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n"
+        },
+        "di" : {
+          "description": "Device Id of VOD",
+          "allOf": [
```

Comment:
Why does "di" have an "allOf" but "n" and "econame" do not?  I'm sure there is a reason but I'm not clear when it's needed and when it's not needed.  Was this a CTT thing?

Reply:
Blame Comarch.  It's because if there is a $ref all other keywords are ignored, it's a wash if just 'description' but a problem if keywords that are 'active' (i.e. readOnly).